src/main/groovy/com/jaredsburrows/license/LicensePlugin.groovy

```diff
@@ -30,6 +30,8 @@ final class LicensePlugin implements Plugin<Project> {
     // Get correct plugin - Check for android library, default to application variant for application/test plugin
     final variants = getAndroidVariants(project)
 
+    final configurationExtension = project.extensions.create("options", AndroidLicenseReportOptions)
```

Comment:
We should probably change `options` to something more clear such as `licenseReport` or just `license`.